### PR TITLE
Reword part about LaTeX and raster images

### LINKED ISSUE #60: Remove part about raster images
"The [...] paragraph about LaTeX users and raster images also seem far-fetched to me. Existing tools to plot profiles generate files in EPS and/or PDF format, which are not of the raster kind, and can easily be included in LaTeX documents."

### PR DESCRIPTION
**Pull request by rgaiacs**
Close #60.

### CODE REVIEW

File: overview.tex [hunk 1]
```diff
@@ -88,8 +88,9 @@ \subsection*{Motivation}
 
     In addition, this tool would allow \LaTeX\ users, a group in which almost
     all optimization community is included, to generate performance profile
-    plots within \LaTeX\ environment, and thus to get a better typographic
-    result other than simple including raster  images.
+    plots as \LaTeX\ code that will be process later
+    with the rest of their document
```
Comment (abelsiqueira) [inline overview.tex]: processed

Comment (rgaiacs) [inline overview.tex]: Fixed.

**Comment by abelsiqueira** [comment]
Sorry, now there is a conflict.

**Comment by rgaiacs** [comment]
> Sorry, now there is a conflict.

Rebased. =)

**Comment by abelsiqueira** [comment]
Thanks

**Pull request merged**